test/unit/Cardano/Wallet/BlockSyncerSpec.hs

```diff
@@ -0,0 +1,113 @@
+{-# LANGUAGE DataKinds #-}
+module Cardano.Wallet.BlockSyncerSpec
+    ( spec
+    ) where
+
+
+import Prelude
+
+
+import Control.Concurrent
+    ( forkIO, killThread, threadDelay, throwTo )
+import Control.Monad.IO.Class
+    ( liftIO )
+import Data.ByteString
+    ( ByteString, pack )
+import Data.IORef
+    ( newIORef, readIORef )
+import qualified Data.Set as Set
+import System.Exit
+    ( ExitCode (..) )
+import Test.Hspec
+    ( Spec, describe, it )
+import Test.Hspec.Expectations
+    ( shouldBe )
+import Test.QuickCheck
+    ( generate, vector )
+import Test.QuickCheck.Gen
+    ( Gen )
+
+
+import Cardano.Wallet.BlockSyncer
+    ( BlocksConsumed (..), blockSyncerBracket, runBlockLayerMockup )
+import Cardano.Wallet.Primitive
+    ( Block (..), BlockHeader (..), Hash (..) )
```

Comment:
Make sure to follow the coding standards for imports :pray:  (two groups, qualified vs explicit imports. We do not make any more groups based on whether the code comes from our code base or not).

Reply:
:+1: